src/Command/ExecCommand.php

```diff
@@ -65,6 +68,14 @@ protected function configure(): void
             'Optional options and arguments to pass to the tool'
         );
 
+        $this->addOption(
```

Comment:
As of 70d01cf18cd19be6cce905c6934ecf51006e5f9d, this should not be needed anymore as it is now unused, the changes in this file should get removed from e458f003855556d434dfc2a64aca074d3970793c.

Reply:
Hm, it was missing when I tried to run the command but looks like it can be skipped now.

Reply:
Fixed with 91f10e0.